ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -2422,6 +2480,24 @@ public sealed class ArchiWebHandler : IDisposable {
 			return (ESteamApiKeyState.Registered, text);
 		}
 
+		private async Task<(ESteamAccessTokenState State, string? Key)> GetAccessTokenState() {
+			Uri request = new(SteamStoreURL, "/pointssummary/ajaxgetasyncconfig");
+
+			ObjectResponse<AccessTokenResponse>? response = await UrlGetToJsonObjectWithSession<AccessTokenResponse>(request).ConfigureAwait(false);
+
+			if (response == null) {
+				return (ESteamAccessTokenState.Timeout, null);
+			}
+
+			if (response.Content.AccessTokenData == null || response.Content.AccessTokenData.Webapi_token == null
```

Comment:
This should probably be `if (string.IsNullOrEmpty(response.Content.AccessTokenData?.Webapi_token)) {`